supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -76,13 +76,23 @@ For more information about Intel® trademarks, see the http://www.intel.com/cont
 ==== image:images/yes.png[yes] Applixware (noun)
 *Description*: "Applixware" is a suite of proprietary modular applications for Linux.
 
-
 *Use it*: yes
 
 *Incorrect forms*: Applix, ApplixWare
 
 *See also*:
 
+[discrete]
+[[as-expected]]
+==== image:images/no.png[no] as expected (adverb)
+*Description*: Expectations are relative, use "correctly" instead.
```

Comment:
Probably should use a semicolon instead of a comma, but otherwise the update looks good to me!

Reply:
I have replaced the comma as you suggested.